README.md

```diff
@@ -163,31 +163,33 @@ Please note that until automatic updates are implemented, any updates released m
 
 #### Updating from the BETA to a RELEASE (1.0.0 onwards)
 
-*Use these instructions if you have a BETA version of JN you are updating to a RELEASE.*
-
-1. **Back up your persistent**: instructions on how to do so may be found [here](https://github.com/Just-Natsuki-Team/NatsukiModDev/wiki/04:-FAQ#can-i-back-up-my-save-data--how-do-i-find-my-persistent).
-2. Download a fresh copy of DDLC from https://ddlc.moe. **Do not use a Steam installation**.
-3. Extract/unzip DDLC.
-4. Download the release: go to https://github.com/Just-Natsuki-Team/NatsukiModDev, then select the most recent release under _Releases_. Select the ZIP file starting with `jn` (I.E `jn-1.0.0.zip`).
-5. Extract/unzip the release files.
-6. **Delete** the `game` and `lib` folders from your **JN installation**.
-7. **Copy** the `game` and `lib` folders from your **fresh DDLC folder** into your JN installation.
-8. From the release files, copy the `game` folder contents (all of it) into the `game` folder of your JN installation. If asked to replace files, select _Yes_.
-9. From the release files, copy the `lib` folder contents (all of it) into the `lib` folder of your JN installation. If asked to replace files, select _Yes_.
-10. From the release files, copy the `update` folder into your JN installation.
-11. Finally, launch `DDLC.exe`.
+*Use these instructions if you have a BETA version of JN you are updating to a RELEASE (1.0.0 onwards).*
+
+1. Stop JN, if you are currently playing the mod.
+2. **Back up your persistent**: instructions on how to do so may be found [here](https://github.com/Just-Natsuki-Team/NatsukiModDev/wiki/04:-FAQ#can-i-back-up-my-save-data--how-do-i-find-my-persistent).
+3. Download a fresh copy of DDLC from https://ddlc.moe. **Do not use a Steam installation**.
+4. Extract/unzip DDLC.
+5. Download the release: go to https://github.com/Just-Natsuki-Team/NatsukiModDev, then select the most recent release under _Releases_. Select the ZIP file starting with `jn` (I.E `jn-1.0.0.zip`).
```

Comment:
```suggestion
5. Download the release: go to [https://github.com/Just-Natsuki-Team/NatsukiModDev](https://github.com/Just-Natsuki-Team/NatsukiModDev), then select the most recent release under _Releases_. Select the ZIP file starting with `jn` (I.E `jn-1.0.0.zip`).
```

Reply:
We can do one better and simply link to the releases: less verbose and quicker to do